src/Prism.DryIoc.Extensions/PrismContainerExtension.cs

```diff
@@ -155,7 +155,8 @@ public IContainerRegistry RegisterManySingleton(Type implementingType, params Ty
 
         public bool IsRegistered(Type type)
         {
-            return Instance.IsRegistered(type);
+            return Instance.IsRegistered(type) ||
+                Instance.IsRegistered(type, factoryType: FactoryType.Wrapper);
```

Comment:
@dadhi is there a better way to get DryIoc to return `true` when querying for `IEnumerable<T>` (or `T[]` or....)?

Reply:
let's keep the conversation going in #124